webcompat/static/js/lib/bugform.js

```diff
@@ -155,6 +156,18 @@ function BugForm() {
     img.src = dataURI;
   };
 
+  // Event handler for image file drop; purpose to detect multiple drops
+  this.handleImageDrop = function() {
+    var imageFilled = false;
+    if ($(".wc-UploadForm-label.is-hidden").length) {
+      imageFilled = true;
+    }
+    if (imageFilled) {
+      // if 2nd drag and drop make wrapper hidden to make remove button available
+      $(".wc-UploadForm-wrapper").addClass("is-hidden");
```

Comment:
Ideally we fix `checkImageTypeValidity` here to not know about form inputs, and to get its data from the `event` object instead.

For the drop event, we can get the file from here: https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/files.